.github/workflows/transformDataToViews.yml

```diff
@@ -0,0 +1,59 @@
+# https://github.com/nvaccess/transformAddonDataToViews/
+
+name: Transform NVDA addons to views
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+    paths:
+      - addons/**
+
+jobs:
+  transformAndPush:
+    environment: transformationAndDeployment
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [ 3.8 ]
+    steps:
+    - name: Checkout addon data
+      uses: actions/checkout@v2
+      with:
+        repository: nvaccess/addon-store-submission
+        path: data
+    - name: Checkout views branch into separate folder
+      uses: actions/checkout@v2
+      with:
+        repository: nvaccess/addon-store-submission
+        path: views
+        ref: views
+    - name: Checkout transformation repo
+      uses: actions/checkout@v2
+      with:
+        repository: nvaccess/transformAddonDataToViews
+        path: transform
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x86
+    - name: Install SSH Client and set deploy key
+      uses: webfactory/ssh-agent@v0.5.3  # https://github.com/webfactory/ssh-agent
+      with:
+        ssh-private-key: ${{ secrets.DEPLOY_KEY }}
```

Comment:
Are you sure this is necessary? The action is run in a "privileged context". It would have been necessary when trying to push from another repo.

See:
- https://github.com/actions/checkout#push-a-commit-using-the-built-in-token
- https://lannonbr.com/blog/2019-12-09-git-commit-in-actions

Reply:
I think you are right, it should be unnecessary in this context.

```suggestion
```